packages/paima-sdk/paima-events/src/app-events.ts

```diff
@@ -0,0 +1,200 @@
+import type { LogEvent, LogEventFields } from './types.js';
+import type { genEvent } from './types.js';
+import { toPath, TopicPrefix } from './types.js';
+import type { TSchema, Static } from '@sinclair/typebox';
+import { Value } from '@sinclair/typebox/value';
+import { keccak_256 } from 'js-sha3';
```

Comment:
TODO: check the size of this in ethereum

Reply:
it's the same size (and not truncated), so if we want to make it shorter it would be different then